types/timestamp_test.go

```diff
@@ -135,3 +135,37 @@ func TestTimestampString(t *testing.T) {
 func utcDate(year, month, day int) time.Time {
 	return time.Date(year, time.Month(month), day, 0, 0, 0, 0, time.UTC)
 }
+
+func BenchmarkStdTimeUnmarshal(b *testing.B) {
+	data, err := StdTimeMarshal(time.Time{})
```

Comment:
What benefit do we get from randomly generated Times? What do you suggest we do with the slice of Times? RunParallel is not necessary and I'll remove.